tool/templates/js2/method.js.jinja

```diff
@@ -5,7 +5,7 @@
     {%- if typescript %}: {{ param.ty -}}{%- endif -%}
     {%- if !loop.last -%}, {% endif -%}
 {%- endfor -%}
-) {%- if typescript %}{{ return_type }};{% else %} {
+) {%- if typescript %}{% if return_type != ": void" %}{{ return_type }}{% endif %};{% else %} {
```

Comment:
nit: in order to have less logic in the template, the return type can be set to `""` in Rust code

Reply:
done